Amalgamoria/Controllers/prompt.py

```diff
@@ -1,5 +1,8 @@
 import random
 from Amalgamoria.Tables.connections import connections
+from Amalgamoria.Tables.modifiers import scene_modifiers
+
+state_team_status_stand_in = 2
 
 class Prompt:
```

Comment:
love me a good class 😊😻 esp. with private access modifiers `_`

Reply:
do you love python even more now that you know the underscore is cosmetic and doesn't actually do anything? 😎

Reply:
no keywords either lol

Reply:
lol (i mean technically its the same in C#.. it doesn't actually _do_ anything)

Reply:
ohh damn i thought this was a unique snake language bit